sources/packages/backend/apps/queue-consumers/src/processors/schedulers/student-application-notifications/student-application-notifications.scheduler.ts

```diff
@@ -39,7 +44,28 @@ export class StudentApplicationNotificationsScheduler extends BaseScheduler<void
         `Processing student application notifications job. Job id: ${job.id} and Job name: ${job.name}.`,
       );
 
-      // TODO: Get applications that have a disability status mismatch  and check PDPPD status
+      const eligibleApplications =
+        await this.applicationService.getEligibleApplicationsForNotification();
```

Comment:
The scheduler will be generic to allow other types of notifications to be generated.
The current notification being worked if related to the student PD/PPD and the methods related to it should express it.
Ideally, the student PD/PPD process of getting pending applications and generating the notifications could be created inside a service and this service be invoked from the scheduler.
Following this would allow to another service be created to process the next application notification when it comes.

Reply:
When other notifications come into place a different query will be created.
We should renamed the `getEligibleApplicationsForNotification` to something specific, for instance, `getApplicationWithPDPPStatusMismatch`.

Reply:
Also, the scheduler should ideally call a service to have its notifications processed.
When adding the next notification process here in the future, we should not expand more logic here.
If the team agrees we can keep this one as it is and refactor when the second notification is added.